addons/info/README.md

```diff
@@ -341,12 +337,11 @@ const TableComponent = ({ propDefinitions }) => {
   );
 };
 
-storiesOf("Button", module).add(
-  "with text",
-  withInfo({
-    TableComponent
-  })(() => <Button>Hello Button</Button>)
-);
+storiesOf('Button', module).add('with text', () => <Button>Hello Button</Button>, {
+  info: {
+    TableComponent,
+  },
+});
```

Comment:
Note that this line is the only actual change in this file, it seems prettier has started formatting it.